pynamodb/tests/test_model.py

```diff
@@ -1925,6 +1925,24 @@ def fake_dynamodb(*args):
             self.assertRaises(UserModel.DoesNotExist, UserModel.get, 'foo', 'bar')
 
         with patch(PATCH_METHOD) as req:
+            req.return_value = {}
+            try:
+                UserModel.get('foo')
+            except SimpleUserModel.DoesNotExist:
+                assert False  # DoesNotExist exceptions must be distinct per-model
```

Comment:
Consider `self.fail('DoesNotExist exceptions must be distinct per-model')` instead